src/Data/PropertyHolder.php

```diff
@@ -136,6 +136,9 @@ public function getPropertyType($propertyName)
             'Crawler',
             'MasterParent',
             'LiteMode',
+            'isFake',
```

Comment:
Nothing. If they not in the ini files, they are ignored, like all full-file-properties when parsing a standard or lite file.